internal/messaging/response_handler.go

```diff
@@ -503,7 +503,7 @@ func createTokenURIforMintResponse(mintResponse *bookv1alpha.MintResponse) (stri
 	attributes := []Attribute{
 		{
 			TraitType: "Mint ID",
-			Value:     mintResponse.GetMintId(),
+			Value:     mintResponse.GetMintId().Value,
```

Comment:
This is because the change of mint id from string to UUID type right?

Reply:
Yes, should I test this part just in case?

Reply:
It's fine